pkg/postage/batchservice/batchservice.go

```diff
@@ -134,7 +134,7 @@ func (svc *batchService) Create(id, owner []byte, totalAmout, normalisedBalance
 		return fmt.Errorf("update checksum: %w", err)
 	}
 
-	svc.logger.Debug("batch created", "batch_id", hex.EncodeToString(batch.ID), "tx", txHash, "tx_checksum", cs)
+	svc.logger.Debug("batch created", "batch_id", hex.EncodeToString(batch.ID), "tx", fmt.Sprintf("%x", txHash), "tx_checksum", fmt.Sprintf("%x", cs))
```

Comment:
using Debugf with `%x` formatter might be better

Reply:
@istae Educate me:  How would that work with the new logger with tag/value pairs?

Reply:
there's no longer a `Debugf` after we transitioned to the new logger...